tensorflow/python/kernel_tests/bias_op_test.py

```diff
@@ -128,7 +128,13 @@ def _testGradient(self, np_input, bias, dtype, data_format, use_gpu):
           input_tensor, np_input.shape, output_tensor, np_input.shape)
       bias_jacob_t, bias_jacob_n = tf.test.compute_gradient(
           bias_tensor, bias.shape, output_tensor, np_input.shape)
-
+         
+      # Test gradient of BiasAddGrad
+      bias_add_grad = tf.gradients(tf.nn.l2_loss(output_tensor),
+                                   bias_tensor)[0]
+      grad_jacob_t, grad_jacob_n = tf.test.compute_gradient(
+          output_tensor, np_input.shape, bias_add_grad, bias.shape)
```

Comment:
Sorry, I was reading through my email backlog, and just saw your update.
Thank you very much for looking into this!